ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -24,7 +24,27 @@ resources:
       memory: 5Gi
 
 tests:
-- as: yaml-lint
-  commands: dist/prow_yaml_lint.sh
-  container:
-    from: src
+  - as: yaml-lint
+    commands: dist/prow_yaml_lint.sh
+    container:
+      from: src
+  - as: rustfmt
+    commands: |
+      rustup component add rustfmt
```

Comment:
to be consistent let's insert `rustup default 1.38.0` before this line too

Reply:
Done